build/NuGetPackages/Microsoft.Build.Localization.nuspec

```diff
@@ -79,5 +79,21 @@
     <file src="$outputPath$tr\Microsoft.Build.Utilities.Core.resources.dll" target="lib\netstandard1.3\tr\Microsoft.Build.Utilities.Core.resources.dll" />
     <file src="$outputPath$zh-Hans\Microsoft.Build.Utilities.Core.resources.dll" target="lib\netstandard1.3\zh-Hans\Microsoft.Build.Utilities.Core.resources.dll" />
     <file src="$outputPath$zh-Hant\Microsoft.Build.Utilities.Core.resources.dll" target="lib\netstandard1.3\zh-Hant\Microsoft.Build.Utilities.Core.resources.dll" />
+
+    <!-- NuGet.MSBuildSdkResolver.dll -->
+    <file src="$outputPath$cs\NuGet.MSBuildSdkResolver.resources.dll" target="contentFiles\any\netstandard1.3\SdkResolvers\NuGet.MSBuildSdkResolver\cs\NuGet.MSBuildSdkResolver.resources.dll"/>
```

Comment:
Put them in lib, like the rest. As far as I remember from my experiments, if you put them in content files, they don't show up in bin when the package is indirectly referenced.

Reply:
I think it is OK as is. What I recall is that they were copied correctly from content files, but not used do to content not going in .deps.json file. But, in this case, the resolver will be loaded dynamically and the msbuild CoreCLR loader will find the satellites from relative path just like tasks.

Reply:
@nguerrera has a better memory than me :)